### DISCUSSION TITLE
Component is Not Dragged

### DISCUSSION THREAD

**Discussion by AgnRakul** [GrapesJS/grapesjs#4798] [Q&A]
I have Added Custom Type for Div and Assigned setDragMode('absolute') While Component is Selected While I try to Drag it The Position: Absolute as been added to the component the property en: 1 is added with strike 
![image](https://user-images.githubusercontent.com/70213104/209277598-e2e7892d-0d35-4f01-a41a-e68f667155f0.png)

But the Component is Not Dragged it like freezed

**Comment by artf** [comment]
Related to this issue: https://github.com/artf/grapesjs/issues/4792
